src/modules/formBuilder/components/formTree/FormTreeItem.tsx

```diff
@@ -79,10 +83,28 @@ const FormTreeItem = React.forwardRef(function FormTreeItem(
               sx={{
                 display: 'flex',
                 flexDirection: 'column',
-                mr: 4,
+                mr: 10,
```

Comment:
This looks pretty silly right now, but I didn't put any time into updating the styles, figuring that we will tackle all styling updates with https://github.com/open-path/Green-River/issues/6093